package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@reactioncommerce/plugin-payments-example",
+  "description": "Example Payments plugin for the Reaction API",
+  "version": "1.0.0",
```

Comment:
Version should be changed to `0.0.0-development", `semantic-release` will take care of bumping the version numbers, it doesn't need to be set here.